### PR TITLE
Initial setup of config files

### PR DESCRIPTION
**Pull request by LilyFirefly**

### CODE REVIEW

**Comment by LilyFirefly** [comment]
@meshy Which license should we use?

**Comment by meshy** [comment]
BSD as per https://github.com/incuna/django-discussion/blob/master/LICENSE

**Comment by LilyFirefly** [comment]
@meshy @KevinEtienne Review/merge?

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from setuptools import setup, find_packages
+from codecs import open
+from os import path
+
+
+here = path.abspath(path.dirname(__file__))
+
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
```
Comment (meshy) [inline setup.py]: As PyPI doesn't support markdown, is there really a point in this?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    author='Incuna',
+    author_email='admin@incuna.com',
+    license='BSD',
+    classifiers=[
+        'Development Status :: 1 - Planning',
+        'Intended Audience :: Developers',
+        'Topic :: Communications',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
```
Comment (meshy) [inline setup.py:21]: Python 3 only?

File: setup.py (hunk 1 above)
Comment (LilyFirefly) [inline setup.py]: Oh - forgot that! I just copied an example `setup.py` that used `rst`.

File: setup.py (hunk 2 above)
Comment (LilyFirefly) [inline setup.py:21]: For now - we can add python 2 support later if we want.

Comment (meshy) [inline setup.py:21]: haha ok... but I was actually asking if you wanted to add:

`'Programming Language :: Python :: 3 :: Only',`

File: setup.py (hunk 1 above)
Comment (meshy) [inline setup.py]: Yeah, please don't use RST!

File: setup.py (hunk 2 above)
Comment (LilyFirefly) [inline setup.py:21]: Ah!

**Comment by LilyFirefly** [comment]
The travis failures can be ignored: `.travis.yml` is added in #1, and we don't have any testable code here.

**Pull request merged**